pkg/controller/operators/olm/requirements_test.go

```diff
@@ -410,14 +410,14 @@ func TestRequirementAndPermissionStatus(t *testing.T) {
 						},
 					},
 				},
-				{"apiextensions.k8s.io", "v1beta1", "CustomResourceDefinition", "c1.g1"}: {
```

Comment:
Why did you need to change these values in the tests?

Reply:
you are right, after some thought, i realize the CRD's gvk should be on the display here not the GVK of the api from that CRD.

Reply:
I was just about to change this back, but then i realized that we are trying to blur the difference between CRD and APIService. In CSVless bundle, we are interpreting all required CRD/APIService without description as a CRD. Therefore, we can no longer, at this stage, tell if that is a CRD or APIService.

Reply:
Got it - so we now defined the GVK of the resource defined in the CRD/APIService. SGTM.